app/src/main/java/com/sdp/movemeet/MainActivity.java

```diff
@@ -42,16 +42,24 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        //phone = findViewById(R.id.text_view_profile_phone);
-        //fullName = findViewById(R.id.text_view_profile_name);
-        //email = findViewById(R.id.text_view_profile_email);
-
-        //handleRegisterUser();
-
 
         ActivitiesUpdater updater = ActivitiesUpdater.getInstance();
         updater.updateListActivities();
 
+        createDrawer();
+
+        handleRegisterUser();
```

Comment:
What is the difference between having this call in the onCreate method and the createDrawer method ?

Reply:
For the moment there is no functional difference, it's just a cleaner way to write it and to avoid a too long onCreate method. The final goal would be to implement this function directly in the Navigation class to avoid duplicating this code in every class that needs a side menu.